docs/montecarlo/sourceintegration.rst

```diff
@@ -0,0 +1,102 @@
+
+Having thus produced attenuated source functions from our Monte Carlo run, we move on to using this to calculate the emerging intensity and finally the luminosity per wavelength. The final integral is given as 
+
+.. math::
+
+   L_\nu  = 8 \pi^2 \int_0^\infty I_\nu (p) p dp
+
+where :math:`p` is the impact parameter of a ray trough the supernova envelope that reaches the distant observer, and :math:`I_\nu (p)` is the intensity along one such ray, given by recursing trough the list of attenuated source functions from the blue to the red and adding up contributions. The relation linking the intensity before the k:th transition :math:`u \rightarrow l` to the intensity after is 
+
+.. math::
+
+   I_k^r = I_k^b e^{-\tau_k} + \left( 1- e^{-\tau_k}\right) S_{k}
+
+where the superscripts are crucial, with :math:`r` and :math:`b` referencing the red and blue sides of the k:th transition respectively. To go from the red side of a line to the blue side of the next we can either ignore continuum sources of opacity, in which case
+
+.. math:: 
+
+   I_{k+1}^b = I_k^r
+
+or include them, then requiring we perform
+
+.. math:: 
+
+   I_{k+1}^b = I_k^r + \Delta \tau_k \left[ \frac 1 2(J_k^r + J_{k+1}^b) - I_k^r  \right]
+
+The starting condition for the blue to red side transition is either :math:`I_0^r = 0` for the case that the impact parameter is greater than the radius if the photosphere, or :math:`I_0^r = B_\nu(T)` if the impact parameter is less than the radius of the photosphere. 
+
+.. note::
+
+   Currently the code does not perform the steps necessary to include continuum sources of opacity.
+
+We seek to integrate all emissions at a certain wavelength :math:`\nu` along a ray at some specific impact parameter :math:`p'`. Because the SNE is expanding homologously, along any ray parallel to the line of sight the Doppler effect will sift a range of co-moving frequencies :math:`\nu_0` into the desired observer frame frequency :math:`\nu`.
+
+To find which lines to include when recursing on the line list we need to find the maximum Doppler shift along a given ray. At any point, the Doppler effect in our coordinates is
+
+.. math::
+
+   \nu = \nu_0 \left( 1 + \beta \mu \right)
+
+where :math:`\beta = \frac v c`, and :math:`\mu = \cos \theta`. Here :math:`\theta` is the angle between the radial direction and the ray to the observer, as shown in the figure below. Because we are in the homologous expansion regime :math:`v = \frac r t`. Solving for :math:`\nu_0` in the above gives the relation we seek, but we require an expression for :math:`\mu`. Examining the figure, we see that for positive :math:`z` the angle :math:`\theta_2` can be related to the :math:`z` coordinate of the point C by
+
+.. math::
+
+   \cos \theta_2 = \frac{z_c}{r} = \mu 
+
+.. image:: https://i.imgur.com/WwVHp5c.png
+
```

Comment:
a and b are not referred. p_c, r_c and z_c are not labeled. Please indicate the direction to the observer.